options/locale/locale_en-US.ini

```diff
@@ -18,34 +18,34 @@ template = Template
 language = Language
 notifications = Notifications
 create_new = Create…
-user_profile_and_more = User profile and more
+user_profile_and_more = Profile and Settings…
 signed_in_as = Signed in as
-enable_javascript = This website works better with JavaScript
+enable_javascript = This website works better with JavaScript.
 
 username = Username
-email = Email
+email = Email Address
 password = Password
-re_type = Re-Type
-captcha = Captcha
-twofa = Two-factor authentication
-twofa_scratch = Two-factor scratch code
+re_type = Re-Type Password
+captcha = CAPTCHA
+twofa = Two-Factor Authentication
+twofa_scratch = Two-Factor Scratch Code
 passcode = Passcode
 
 repository = Repository
-organization = Organization
+organization = Group
```

Comment:
@techknowlogick I think you missed parts of the diff. The hierarchy is expressed as 'group' and 'subgroup'. Hierarchies don't get any clearer than that in English.

@lunny : the main reason @techknowlogick didn't like it ('no implied hierarchy') is actually debunked. Do you have any other reasons to want to keep Organization?

Ten days ago I actually motivated this change in #3512, but got no replies except for a positive from @thehowl. Unless my statements there are incorrect I see no reason to switch back to Organization and Team, which are (in my opinion) unclear and unfit for Gitea's small-scale, self-hosted use case.

Reply:
@bugreport0 The main reason I mentioned for my hesitance with this change is that "organization" is used as a reference in code, as well as in API paths. Using org/team language aligns with GitHub as well (especially as we are trying to have an API that aligns with theirs).

Looking at the ticket you referenced, @thehowl makes references to how GitLab does it, however as mentioned we are trying to align with GitHub.

There are many different use cases for Gitea, it sounds like in yours it is small scale, however there are users who use Gitea to support multiple different orgs, and a large number of users.

Reply:
I also do like group better than org but that is quite big change and should be probably done in either other PR or issue so that it does not block other changes here

Reply:
@lafriks OK, let's try it that way I'll be updating my PR in a jiffy.